app/assets/stylesheets/footer.scss

```diff
@@ -0,0 +1,36 @@
+$clir-burgundy: #560319;
+$dlf-blue:      #046BA5;
+$bg-gray:       #eef2f5;
+
+
+@import 'https://fonts.googleapis.com/css?family=Crimson+Text:600';
+@import 'https://cdn.rawgit.com/clirdlf/logo-fonts/master/style.min.css';
+@import 'https://cdn.rawgit.com/clirdlf/logo-fonts/master/clir-font/stylesheet.min.css';
+@import 'https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css';
```

Comment:
This is going to add like about 200k to the page weight just for 3 icons? Are you sure?

Reply:
Do these domains need to be added to the CSP headers in `config/initializers/secureheaders.rb`?

Reply:
By making these `@import` aren't we loosing any benefit that a cdn would bring?

Reply:
I think we should make a note to revisit these questions later. I certainly share your concern about these being a little heavy-handed for 3 social media icons and 2 logos, but 🤷‍♂️